src/Tools/Common/Commands/ProcessStatus.cs

```diff
@@ -174,8 +173,16 @@ void FormatTableRows(List<ProcessDetails> rows, StringBuilder tableText)
                 FormatTableRows(printInfo, sb);
                 console.Out.WriteLine(sb.ToString());
             }
-            catch (InvalidOperationException ex)
+            catch (Exception  ex)
             {
+                if (ex is InvalidOperationException)
+                {
+                    console.Out.WriteLine(ex.ToString());
+                }
+                else if (ex is NamedPipeEnumerationUnauthorizedException)
+                {
+                    console.Out.WriteLine(ex.ToString());
+                }
                 console.Out.WriteLine(ex.ToString());
             }
```

Comment:
You'll end up writing the exception twice here. If we are going to just log the exception we should probably just:

```suggestion
            catch (Exception  ex)
            {
                console.Out.WriteLine(ex.ToString());
            }
```

In general, this makes things hard to use for automation in some sense, since you won't get an exit code from the tool, but that's another preexisting issues (although it used to crash due to an exception and now it doesn't.